docs/conf.py

```diff
@@ -234,3 +245,217 @@ def setup(app):
+            % pdfpagelayout=SinglePage,
+            % pdfpagelabels=true,
+            % bookmarksopen=true,
+            % bookmarksopenlevel=0
+        }
+        \makeatother
+
+        %%%  COVER PAGE  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+        \pagenumbering{Roman} %%% to avoid page 1 conflict with actual page 1
+
+        \begin{titlepage}
+            \centering
+
+            \vspace*{80mm} %%% * is used to give space from top
+            \begin{figure}[!h]
+            \centering
+            \includegraphics[scale=0.75]{logo.png}
+            \end{figure}
+            
+            \vspace{0mm}
+            \textbf{\Huge {Kolibri User Guide}}
+
+            \vspace{0mm}
+            \small Published by Learning Equality
+
+            %% \vfill adds at the bottom
+            \vfill
+            \small \textit{More information available at }{\href{https://learningequality.org}{learningequality.org}}
+        \end{titlepage}
+
+        \clearpage
+        \pagenumbering{roman}
+        \tableofcontents
+        % \listoffigures
+        % \listoftables
+        \clearpage
+        \pagenumbering{arabic}
+        ''',
+    # Latex figure (float) alignment
+    # 'figure_align': 'htbp',
+    #
+    # GLOBAL OPTIONS FOR THE sphynx.sty STYLE CLASS ############################
+    'sphinxsetup': \
+        'hmargin={1in,1in}, vmargin={1.2in,0.7in}, \
+        TitleColor={rgb}{0,0,0}, \
+        InnerLinkColor={rgb}{0,0,1}, \
+        OuterLinkColor={rgb}{0,0,1}, \
+        verbatimwithframe=false, \
+        VerbatimColor={rgb}{0.95,0.95,0.95}, \
+        verbatimvisiblespace={}, \
+        verbatimcontinued={}', 
+        'tableofcontents':' ',
+    }
+    # other sphinxsetup options:
+    #       verbatimwithframe=true, \
+    #       VerbatimColor={named}{OldLace}, \
+    #       verbatimsep=3pt
+    #       hintBorderColor={named}{LightCoral},
+    #
+    #       attentionborder=3pt,
+    #       attentionBorderColor={named}{Crimson},
+    #       attentionBgColor={named}{FloralWhite},
+    #
+    #       noteborder=2pt,
+    #       noteBorderColor={named}{Olive},
+    #
+    #       cautionborder=3pt,
+    #       cautionBorderColor={named}{Cyan},
+    #       cautionBgColor={named}{LightCyan}}
+    ############################################################################
+
+
+
+# Show URLs in footnote
+latex_show_urls = 'footnote'
+
+# Grouping the document tree into LaTeX files. List of tuples:
+latex_documents = [
+    ( 'index',                              # source start file,
+      'kolibri.tex',                        # target name,
+     u'Kolibri User Guide',                 # title,
+     u'Published by learningequality.org',  # author,
+      'manual'                              # documentclass [howto, manual, or own class])
+     ),
+]
+
+# The name of an image file (relative to this directory) to place at the top of
+# the title page.
+#latex_logo = None
+latex_logo = 'logo.png'
```

Comment:
this is probably not used since we do custom cover-page in `maketitle`

Reply:
I think that's what specifies which file gets displayed in the cover page, right?

Reply:
If we were to use the default cover page (provided by basic sphinx class) then it would use the logo specified here, but since we define a custom cover page,
https://github.com/learningequality/kolibri-docs/blob/develop/docs/conf.py#L339-L390
the `latex_logo` option is not used.

@radinamatic  If you want to change the cover image change this line: https://github.com/learningequality/kolibri-docs/blob/develop/docs/conf.py#L369
replacing with some high resolution pdf of the .svg khan sent